File: Wisdom-Knowledge/Harmenszoon/Harmenszoon.java
public class Harmenszoon {

    private String role = "Warlock";
    private String lane = "Mid Lane";
    private int mana = 1000;
    private int health = 1500;

    public void useQ() {
        System.out.println("Casting Distortion: Confuses the enemy!");
    }

    public void useW() {
        System.out.println("Casting Disfunction: Enemy attacks itself!");
    }

    public void useE() {
        System.out.println("Casting Dehydrate: Slows and drains vitality.");
    }

    public void useR() {
        System.out.println("Casting Desiderata: Removes will to fight!");
    }

    public void showStatus() {
        System.out.println("Role: " + role + ", Lane: " + lane);
        System.out.println("Mana: " + mana + ", Health: " + health);
    }

    public static void main(String[] args) {
        Harmenszoon harmenszoon = new Harmenszoon();
        harmenszoon.showStatus();
        harmenszoon.useQ();
        harmenszoon.useW();
        harmenszoon.useE();
        harmenszoon.useR();
    }
}

File: Wisdom-Knowledge/Harmenszoon/harmenszoon.py
class Harmenszoon:
    def __init__(self):
        self.role = "Warlock"
        self.lane = "Mid Lane"
        self.mana = 1000
        self.health = 1500

    def use_q(self):
        print("Casting Distortion: Confuses the enemy!")

    def use_w(self):
        print("Casting Disfunction: Enemy attacks itself!")

    def use_e(self):
        print("Casting Dehydrate: Slows and drains vitality.")

    def use_r(self):
        print("Casting Desiderata: Removes will to fight!")

    def show_status(self):
        print(f"Role: {self.role}, Lane: {self.lane}")
        print(f"Mana: {self.mana}, Health: {self.health}")

if __name__ == "__main__":
    harmenszoon = Harmenszoon()
    harmenszoon.show_status()
    harmenszoon.use_q()
    harmenszoon.use_w()
    harmenszoon.use_e()
    harmenszoon.use_r()
